src/features/Common/data/api.js

```diff
@@ -1,13 +1,13 @@
 import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 import { getConfig } from '@edx/frontend-platform';
 
-function getCoursesByInstitution(institutionId, limit, page, filters) {
+function getCoursesByInstitution(institutionId, limit, page, filters, licenseId) {
```

Comment:
Maybe add another parameter is not necessary, you got the `filters` params ( maybe the name is not descriptive ) but you could used, aflter all when the request is maded it use a

```js
{ params }
``` 
object and the request url will look like:

```js
const filters = {
    license_id: "example123"
}

// url output
// /courses/?limit=<limit>&institution_id=<id>&license_id=example123
 
```

Lemme now if I was clear I think I made the explanation little confusing.

Reply:
@01001110J It's so clear!
I'll implement it as you mentioned.